google-cloud-nio/src/main/java/com/google/cloud/storage/contrib/nio/CloudStorageFileSystem.java

```diff
@@ -61,10 +62,11 @@
   public static final int BLOCK_SIZE_DEFAULT = 2 * 1024 * 1024;
   public static final FileTime FILE_TIME_UNKNOWN = FileTime.fromMillis(0);
   public static final Set<String> SUPPORTED_VIEWS = ImmutableSet.of(BASIC_VIEW, GCS_VIEW);
-
   private final CloudStorageFileSystemProvider provider;
   private final String bucket;
   private final CloudStorageConfiguration config;
+  private static final Map<CloudStorageConfiguration, CloudStorageFileSystemProvider>
+      FILE_SYSTEM_CONFIG = new HashMap<>();
```

Comment:
FILE_SYSTEM_CONFIG is not an informative name for a variable storing map config to provider

Reply:
Done